pylutron/__init__.py

```diff
@@ -825,21 +825,23 @@ class Event(LutronEvent):
               generate this event.
         Params: None
     HOLDED:
-    TAPPED:
+    DOUBLETAPPED:
     HOLD_RELEASED    
     """
     PRESSED = 1
     RELEASED = 2
     HOLDED = 3
-    TAPPED = 4
+    DOUBLETAPPED = 4
     HOLD_RELEASED = 5
 
 
-  def __init__(self, lutron, keypad, name, num, button_type, direction):
+  def __init__(self, lutron, keypad, name, engraving, num, button_type, direction, led_logic):
```

Comment:
line too long..

Also, what is led_logic? You added it in the other change, and needless to say you need to merge these two together, but what does led_logic mean? where do you use it?

Reply:
Led state can be
0 = Off  
1 = On 
2 = Normal-Flash: 1 flash every second 1  
3 = Rapid-Flash: 10 flashes every second 1

Previous code was using true/false only. Maybe it's radiora2?